### DISCUSSION TITLE
Wiki.js isn't working after initial setup and redirect

### DISCUSSION THREAD

**Discussion by Phreeak** [tteck/Proxmox#907] [Help Needed]
Hey, i tried your script for wiki.js. I can successfully setup the lxc container. I can also doing the initial setup via webbrowser. 
But after setting password and domainname, the redirection (ip:3000) doesn't work :/

Could you please check these?

Many thanks for your help! You do an awesome job

**Comment by tteck** [comment]
https://docs.requarks.io/install/config

path to config.sample.yml `/opt/wikijs/config.sample.yml` (needs renamed)
path to db `/opt/wikijs/db.sqlite`
path to data `/opt/wikijs/data`

That's all I know

**Reply by tteck** [reply]
I can have the script copy the `config.sample.yml` to `config.yml` and make some changes, if I knew what changes were needed.
So, when you get it going, share your config and I'll make the changes.
